x_and_o/x_and_o.c

```diff
@@ -0,0 +1,163 @@
+#include <stdio.h>
+#include <errno.h>
+#include "x_and_o.h"
+
+void initial_board(char board[BOARD_SIZE][BOARD_SIZE])
+{
+    for (size_t rows = 0; rows < sizeof(*board); rows++)
```

Comment:
I would have used BOARD_SIZE instead sizeof()..
It is more understandable and clear.